view_all_set.php

```diff
@@ -374,11 +94,6 @@
 	$f_per_page = config_get( 'default_limit_view' );
```

Comment:
marked to remove in next clean-up commit.
that code s not needed as it is moved to filter_gpc_get()